docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -690,6 +712,10 @@ which can be modified with `UserNS`, but if that is not specified, this UID is a
 Set the user namespace mode for the container. This is equivalent to the Podman `--userns` option and
 generally has the form `MODE[:OPTIONS,...]`.
 
+### `VolatileTmp=`
+
+FIXME
```

Comment:
I would need to look through the git log but if I remember correctly this key is deprecated and thus not documented.
I like the idea of this but I think we need to have a way to ignore some keys. Maybe we could add `// deprecated` as comment in the code behind the key name and you script could see it and not force the docs or something like this.

Reply:
Thank you! #20479. I had missed that. And I like the idea of `// deprecated`.